Region/MinMax.h

```diff
@@ -43,10 +43,10 @@ void MinMax<T>::operator()(const tbb::blocked_range<size_t> &r) {
     T tmax = maxval;
     for(size_t i = r.begin(); i != r.end(); ++i) {
         T val = data[i];
-	if(std::isfinite(val)) {
-	    if(val < tmin) {
+	if( std::isfinite(val) ) {
```

Comment:
I'm pretty sure she meant `if (std::isfinite(val)) {`, not `if( std::isfinite(val) ) {`